src/common/libmmgcommon_private.h

```diff
@@ -130,6 +130,23 @@ LIBMMG_CORE_EXPORT int  MMG5_Set_outputMeshName(MMG5_pMesh mesh, const char* mes
  *
  */
 LIBMMG_CORE_EXPORT int  MMG5_Set_inputSolName(MMG5_pMesh mesh,MMG5_pSol sol, const char* solin);
+/**
+ * \param mesh pointer toward the mesh structure.
+ * \param fparamin name of the input parameter file.
+ * \return 0 if failed, 1 otherwise.
+ *
+ *  Set the name of output solution file.
+ *
+ * \remark Fortran interface:
+ * >   SUBROUTINE MMG5_SET_OUTPUTSOLNAME(mesh,fparamin,strlen,retval)\n
```

Comment:
Wrong doc copy-paste ;-)). `OUTPUTSOL` has to be replaced by `INPUTPARAM`.

Updating this specific part of doc really matters: it is used to generated automatically the Fortran interface.